subsys/usb/class/audio/audio.c

```diff
@@ -34,6 +34,10 @@ struct usb_audio_dev_data {
 
 	uint8_t ch_cnt[2];
 
+	uint8_t frequency[2][3];
+
+	uint32_t *frequencies[2];
```

Comment:
The 2 is because there are two channels? Could you please add a define or constant for that?

Reply:
Changed it to uint32_t which is much better when using the sys_get_le24/sys_put_le24 functions.
Also renamed it to frequency_selected.
the constant is not clear, I just continued with the current way of using it. See https://github.com/zephyrproject-rtos/zephyr/pull/26771#discussion_r452814674
for discussion on changes regarding input/output and what is called device.

Reply:
We can `#define MAX_LOGICAL_DEVICES 2` in `usb_audio_internal.h` if You like @jfischer-phytec-iot .

Reply:
Yes please.